filewriter_v2/README.md

```diff
@@ -0,0 +1,388 @@
+
+| Parameter | JSON Type | Default | Description |
+|-----------|------|---------|-------------|
+| `enabled` | boolean | `false` | Enabled state of the FileWriter. |
+| `image_id_offset` | uint | `1` | First image number on the axis `/entry/image_number`. |
+| `name_pattern` | string | `series_$id` | Basename of the files. `$id` will be replaced by the series number, i.e. using the default, the generated files will have the names:<br> `series_<series_number>_master.h5`<br> `series_<series_number>_data_<file_number>.h5`|
+| `images_per_file` | uint | `1000` | Maximum number of images stored in each data file. If set to `0`, all images are stored directly in the master file and no data files are created. <br> If set to a value greater than zero, the images are stored in multiple data files, but are still accessible in the masterfile via the dataset "/entry/data/data", since this is a [Virtual Dataset](https://portal.hdfgroup.org/display/HDF5/Virtual+Dataset++-+VDS) in this case.|
+| `v2_preview_enabled` | bool | `false` | Enabled state of the FileWriter V2 preview. |
+-->
+
+
+### Example
+
+FileWriter V2 may be configured with cURL.
+
+Using SIMPLON:
+```sh
+curl \
+    -X PUT \
+    -H "Content-Type: application/json" \
+    -d '{"mode": {"value": "enabled"}, "v2_preview_enabled": {"value": true}}' \
+    $ADDRESS_OF_DCU/filewriter/api/1.8.0/config
+```
+
+## Data Access
+
+The files are stored locally on the DCU and have to be transferred to the user computer, this can also be done via cURL using the SIMPLON API.
+
+```sh
+# list available files
+curl $ADDRESS_OF_DCU/filewriter/api/1.8.0/status/files
+
+# file access
+curl $ADDRESS_OF_DCU/data/<name_pattern>_master.h5
+curl $ADDRESS_OF_DCU/data/<name_pattern>_data_<file_number>.h5
+```
+
+For further details on available commands and status parameters please refer to the [SIMPLON API documentation](https://media.dectris.com/210607-DECTRIS-SIMPLON-API-Manual_EIGER2-chip-based_detectros.pdf).
+
+## Multi-channel Data
+
+One of the main new features of FileWriter V2 is the support of multi-channel data, i.e. each image consists of multiple channels, one per enabled threshold/difference mode. Therefore, the dataset `/entry/data/data` is four-dimensional, i.e. `[number_of_images, number_of_channels, width, height]`. The channels are identified via strings given in the [_AXISNAME_ field](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-field) `channel` of the _NXdata_ class[^2]. For every channel name listed in this field, there exists a subgroup of the [_NXdetector_]() group of type _NXdetector_channel_[^1] with the same name followed by the suffix `_channel`. This group contains channel-specific metadata, for example `threshold_energy`, `flatfield` or `pixel_mask`.
+
+An _NXdetector_channel_ group could contain every field that is allowed for NXdetector and that makes sense to be given per channel.
```

Comment:
"A" group

Reply:
isn't it "an" because of the "NX" (which is pronounced "eneks"?)

Reply:
I think "an" is correct.

Reply:
this was interesting to find out:

```
A and an are two different forms of the same word: the indefinite article a that is used before noun phrases. Use a when the noun or adjective that comes next begins with a consonant sound. Use an when the noun or adjective that comes next begins with a vowel sound. Remember that what matters is the pronunciation, not the spelling.
```
https://www.britannica.com/dictionary/eb/qa/how-do-you-know-whether-to-use-a-or-an

Reply:
the spellchecker here, doesn't know that, and just suggest A because the word starts with N.

Reply:
I'll leave it as is then :)